openlayers/index.d.ts

```diff
@@ -11695,7 +11695,7 @@ declare module olx {
             resolution: number;
             start?: number;
             duration?: number;
-            easing: ((t: number) => number);
+            easing?: ((t: number) => number);
```

Comment:
This change (and all the following 'easing' changes) have been done in a more recent version of ol (v3.20.0). Please update the version of the definition file accordingly.